js/app.js

```diff
@@ -23,7 +23,7 @@ function TextApp() {
 TextApp.prototype.init = function() {
   this.settings_ = new Settings();
   this.analytics_ = new Analytics();
-  this.editor_ = new Editor($('#editor')[0], this.settings_);
+  this.editor_ = new EditorCodeMirror($('#editor')[0], this.settings_);
```

Comment:
can this be moved to the initEditor method in an if/else on accessiblity mode? that way we can remove the need for the to call removeEditor on startup

Reply:
done!